modules/tensor_mechanics/include/scalarkernels/GeneralizedPlaneStrainDiag.h

```diff
@@ -0,0 +1,32 @@
+/****************************************************************/
```

Comment:
I think this should just be called `GeneralizedPlaneStrain` because it computes the residual corresponding to the scalar variable.